servo/runner.py

```diff
@@ -234,7 +271,13 @@ def _reraise_if_necessary(task: asyncio.Task) -> None:
         self._main_loop_task = asyncio.create_task(self.main_loop(), name=f"main loop for servo {self.optimizer.id}")
         self._main_loop_task.add_done_callback(_reraise_if_necessary)
 
-    async def run(self, *, poll: bool = True) -> None:
+        if diagnostics:
```

Comment:
Please move diagnostic task init to the `run` method just above this line https://github.com/opsani/servox/blob/e5d744cb298f6a93d1c55354ba6300f246be67e8/servo/runner.py#L325